src/test/java/gov/healthit/chpl/aqa/asserts/ListingHistoryEyeAsserts.java

```diff
@@ -54,7 +54,8 @@ public void iSeeListingHistory(final String expectedEditDetails) throws Exceptio
         }
         Date actualDate = getFormatedDateTimeZone(latestDate);
         Date currentDate = new Date();
-        assertTrue(((currentDate.getTime() - actualDate.getTime()) < TIME_DURATION), "Listing history is missing for today's date");
+        assertTrue(((currentDate.getTime() - actualDate.getTime()) < TIME_DURATION),
+                "Listing history is missing for today's date");
```

Comment:
I know you didn't change the contents, but the text here doesn't match the assertion